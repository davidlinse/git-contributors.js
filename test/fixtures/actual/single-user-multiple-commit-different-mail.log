John Doe john@doe.com
John Doe john@doe.com
John Doe john@doe.com
John Doe john@doe.com
John Doe john@doe.com
John Doe john@doe.com
John Doe john@doe.org
John Doe john@doe.org
John Doe john@doe.org
John Doe john@doe.org
