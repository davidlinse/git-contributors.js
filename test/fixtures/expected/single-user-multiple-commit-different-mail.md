+ John Doe <john@doe.com>  (6 - 60%)
+ John Doe <john@doe.org>  (4 - 40%)
