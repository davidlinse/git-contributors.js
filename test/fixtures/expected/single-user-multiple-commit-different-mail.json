[
  {
    "commits": 6,
    "email": "john@doe.com",
    "name": "John Doe",
    "percent": 60
  },
  {
    "commits": 4,
    "email": "john@doe.org",
    "name": "John Doe",
    "percent": 40
  }
]
